app/components/forms/wizard/sponsors-step.js

```diff
@@ -54,19 +54,8 @@ export default Component.extend(FormMixin, {
     },
     saveDraft() {
       this.onValid(() => {
-        let { sponsors } = this.data;
-        let incorrect_sponsors = sponsors.filter(function(sponsor) {
-          return (!sponsor.get('name'));
-        });
-        if (incorrect_sponsors.length > 0) {
-          this.notify.error(this.l10n.t('Please fill the required fields.'), {
-            id: 'req_field_spon'
-          });
-          this.set('isLoading', false);
-        } else {
-          this.set('data.event.state', 'draft');
-          this.sendAction('save');
-        }
```

Comment:
Why remove this?

Reply:
when the data is validated then there is no need for rest. This part is supposed to be in addSponser function only.

Reply:
So is it present there?

Reply:
yes it is present there.